code/drasil-theory/lib/Theory/Drasil/MultiDefn.hs

```diff
@@ -45,41 +44,39 @@ makeLenses ''MultiDefn
 
 
 instance HasUID        (MultiDefn e) where uid     = rUid
-instance HasSymbol     (MultiDefn e) where symbol  = symbol . (^. qd)
 instance NamedIdea     (MultiDefn e) where term    = qd . term
 instance Idea          (MultiDefn e) where getA    = getA . (^. qd)
 instance HasSpace      (MultiDefn e) where typ     = qd . typ
-instance Quantity      (MultiDefn e) where
 instance MayHaveUnit   (MultiDefn e) where getUnit = getUnit . view qd
 -- | The concept domain of a MultiDefn is the union of the concept domains of the underlying variants.
 instance ConceptDomain (MultiDefn e) where cdom    = foldr1 union . NE.toList . NE.map (^. cd) . (^. rvs)
 instance Definition    (MultiDefn e) where defn    = rDesc
 -- | The complete Relation of a MultiDefn is defined as the quantity and the related expressions being equal
 --   e.g., `q $= a $= b $= ... $= z`
 instance Express e => Express (MultiDefn e) where
-  express q = equiv $ sy q : NE.toList (NE.map (express . (^. expr)) (q ^. rvs))
+  express q = equiv $ sy (q ^. qd) : NE.toList (NE.map (express . (^. expr)) (q ^. rvs))
 
 -- | Smart constructor for MultiDefns, does nothing special at the moment. First argument is the 'String' to become a 'UID'.
-mkMultiDefn :: String -> QuantityDict -> Sentence -> NE.NonEmpty (DefiningExpr e) -> MultiDefn e
+mkMultiDefn :: UID -> QuantityDict -> Sentence -> NE.NonEmpty (DefiningExpr e) -> MultiDefn e
 mkMultiDefn u q s des
-  | length des == dupsRemovedLen = MultiDefn (mkUid u) q s des
+  | length des == dupsRemovedLen = MultiDefn (u +++. "MD") q s des
```

Comment:
Although this embedded string does occur only once, the same pattern should be followed where this doesn't happen inline, but through a single spot.